person.rb

```diff
@@ -1,3 +1,5 @@
+require_relative '../decorators/nameable'
+
 class Person
```

Comment:
- [ ] the person class should inherit from the nameable class and should have a method correct name.